ansible/roles/accumulo/templates/hadoop-metrics2-accumulo.properties

```diff
@@ -20,3 +20,9 @@ accumulo.sink.graphite.class=org.apache.hadoop.metrics2.sink.GraphiteSink
 accumulo.sink.graphite.server_host={{ groups['metrics'][0] }}
 accumulo.sink.graphite.server_port=2004
 accumulo.sink.graphite.metrics_prefix=accumulo
+
+*.sink.statsd.class=org.apache.hadoop.metrics2.sink.StatsDSink
```

Comment:
What is the purpose of this? Is this change meant for EC2 and Azure?

Reply:
Thanks for the question, Keith. The [Azure Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-sources-collectd) agent can send collectd metrics to [Azure Monitor Logs](https://docs.microsoft.com/en-us/azure/azure-monitor/). Given there is Hadoop metrics2 support for a statsd sink, and given there is a statsd plugin for collectd, it allows us to send metrics from Accumulo, Hadoop and Zookeeper through to Azure Monitor. 

This change is therefore specifically to support our changes for cluster_type == azure. If you prefer that we make this conditional, that's definitely something we can do.

Reply:
@keith-turner this is an extra sink for statsd and does not break the existing integration with Influx / Grafana. We'll make the statsd sink conditional for cluster_type == 'azure' to keep it clean.

Reply:
Were you thinking of doing that in the PR or as follow on work?

Reply:
Disregard previous comment.  Resolving this because moving the stats specific changes to Azure section of code is being discussed in another conversation thread.